lib/assets/javascripts/cartodb/table/header/options_menu.js

```diff
@@ -42,6 +43,7 @@ cdb.admin.HeaderOptionsMenu = cdb.admin.DropdownMenu.extend({
     this.user = this.options.user;
     this.dataLayer = this.options.dataLayer;
     this.table = this.dataLayer.table;
+    this.vis = this.options.vis;
```

Comment:
Whoops. I'll update the new functions to use `this.vis`. It helps to see at a glance what the required options are for the view.